packages/shared/src/hooks/useLogin.ts

```diff
@@ -97,12 +98,17 @@ const useLogin = ({
     onPasswordLogin({ action, params });
   };
 
-  useWindowEvents('message', async (e) => {
-    if (!e.data?.flow) {
+  useKeyedWindowEvent('message', AuthEvent.Login, async () => {
```

Comment:
I'm still pretty sure this hook doesn't have to have this window event.

Reply:
Once you finish the social provider login it will redirect you to our callback which this is our way of letting the tab know it has finished.

Reply:
I haven't seen this being fired, is it only with the login flow?

Reply:
The event is being called from here in the callback page.

https://github.com/dailydotdev/apps/blob/8b064d112a8471803b0ba87f437d9d14b6eb69b9/packages/webapp/pages/callback.tsx#L5-L17

Reply:
Oh, wait. I haven't pushed a config in heimdall to make this work 🤦